File: Python/2356.py
#2356 - Bactéria I
# -*- coding: utf-8 -*-

def main():
    while(True):
        try:
            a = str(input())
        except EOFError:
            break;
        b = str(input())

        if(b in a):
            print("Resistente")
        else:
            print("Nao resistente")


if __name__ == '__main__':
    main()

File: C/2356.c
//2356 - Bactéria I
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main(){
    char dna[105], test[105];
    char *controle;

    while (scanf("%s", dna) != EOF) {
        scanf("%s", test);

        controle = strstr(dna, test);

        if(!controle){
            printf("Nao resistente\n");
        }
        else{
            printf("Resistente\n" );
        }
    }

    return 0;
}
